.github/workflows/full-site-editing-plugin.yml

```diff
@@ -1,8 +1,9 @@
 on:
   pull_request:
-    # only trigger this workflow if FSE plugin files have been modified
+    # only trigger this workflow if FSE plugin files have been modified, or if packages have been updated.
     paths:
-    - 'apps/full-site-editing/full-site-editing-plugin/**'
+    - 'apps/full-site-editing/**'
```

Comment:
Not sure if we should build and sync diff creation for things outside of actual plugin folder. 🤔 It probably won't cause issues since we only merge version bump PRs anyway. Another option would be to isolate the changes here in a separate workflow file with these broader matchers.

Reply:
That might be possible. IMO, we should be running:
- `yarn build`
- `phpunit`
- `phpcs`

to any change in apps/full-site-editing (e.g. to pick up on config changes). And on changes to yarn-lock. 

IMO the sync diff is the exception 🤔 I wonder if we can restrict a job based on the path itself.

Reply:
> Not sure if we should build and sync diff creation for things outside of actual plugin folder.  It probably won't cause issues since we only merge version bump PRs anyway.

Yeah, and there's currently not a lot of files in `apps/full-site-editing/` (outside of `apps/full-site-editing/full-site-editing-plugin`), so the overhead shouldn't be too bad.

> Another option would be to isolate the changes here in a separate workflow file with these broader matchers.

That might make sense in the long run.

Personally, I wouldn't block merging this PR on that -- I think it's important to get PHPCS into CI, and we can tweak and iterate _how_ we do it later.

Reply:
> IMO the sync diff is the exception 🤔 I wonder if we can restrict a job based on the path itself.

Afaik that hasn't been introduced in GH actions proper yet, although I've seen some 3rd party actions for it (I would rather avoid that dependency).

Reply:
I don't consider this a blocker either, just a possible future improvement.